starter-code/index.js

```diff
@@ -1,10 +1,29 @@
-var SortedList = function() {};
-
-SortedList.prototype.add     = function(item) {}
-SortedList.prototype.get     = function(pos) {}
-SortedList.prototype.max     = function() {}
-SortedList.prototype.min     = function() {}
-SortedList.prototype.average = function() {}
-SortedList.prototype.sum     = function() {}
+class SortedList {
+  constructor() {
+    this.items = [];
+    this.length = this.items.length;
+  }
+  add (item){
+    this.items.push(item);
```

Comment:
Este test no pasaba, porque al final del método necesitabas igualar `this.length` a la longitud de los items.

`
this.length = this.items.length;
`